addons/hunterkiller/stringtable.xml

```diff
@@ -36,6 +38,7 @@
             <German>Richtet deinen Turm auf die Blickrichtung eines anderen Turms</German>
             <Korean>포탑을 다른 포탑의 시야에 놓습니다.</Korean>
             <Japanese>自分のタレットを相手のタレットの視点に旋回させる。</Japanese>
+            <Chinesesimp>将你的炮塔转向至另一个炮塔正在观察的位置</Chinesesimp>
```

Comment:
these seem to be the referenced lines, feel free to make a replacement suggestion

Reply:
Its in another line, changed it down below